CHANGELOG.rst

```diff
@@ -11,8 +11,17 @@ The rules for MAICoS' CHANGELOG file:
 
 .. inclusion-marker-changelog-start
 
-v0.10 (2025/05/12)
+Unreleased
 ----------
+Philip Loche
+
+- Move changelog test from tox to Gitlab CI (#483)
+- Move from Gitlab to GitHub (#1)
+
+.. Note: moved to github after 0.10; `!XXX` links are not valid anymore.
```

Comment:
The !xxx numbers are not links on github anyway, so maybe just add info on where to find them?

```suggestion
.. Note: moved from gitlab to github after 0.10; `!XXX` reference gitlab merge requests at https://gitlab.com/maicos-devel/maicos/-/merge_requests
```